common/exprParser/expression.js

```diff
@@ -3,10 +3,9 @@ const R = require('ramda')
 const jsep = require('./helpers/jsep')
 const { evalExpression } = require('./helpers/evaluator')
 const { toString: toStringUtils, isValid } = require('./helpers/utils')
-const { types } = require('./helpers/types')
+const { types } = require('./types')
```

Comment:
there's no need to move types. it was intentionally left inside the private 'helpers' folder.
It is exported from Expression. So to access it from outside, you can use  ```Expression.types```

Reply:
ok